bot-review/action.yml

```
@@ -0,0 +1,22 @@
+name: 'Bot review'
+description: 'Reviews a PR in name of exivity-bot'
```

I think user depends on token passed in?
```suggestion
description: 'Reviews a PR'
```